src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+body {
```

Comment:
it's not part of `card` BEM block, so shouldn't be here
you may create a separate file for it